AK/NumericLimits.h

```diff
@@ -101,23 +101,23 @@ struct NumericLimits<unsigned long long> {
 #ifndef KERNEL
 template<>
 struct NumericLimits<float> {
-    static constexpr float min() { return __FLT_MIN__; }
+    static constexpr float min() { return -__FLT_MAX__; }
```

Comment:
Assuming we want these to match std::numeric_limits, this change is incorrect.

https://en.cppreference.com/w/cpp/types/numeric_limits/min
https://en.cppreference.com/w/cpp/types/numeric_limits/lowest

min() is supposed to be __FLT_MIN__, while -__FLT_MAX__ is lowest()

Reply:
That is a horrifying API, was not aware of `lowest()` vs `min()` in floating-point types.

https://en.cppreference.com/w/cpp/types/numeric_limits/min
>  Note that this behavior may be unexpected

Ya don't say!

Fixed this by adding `lowest()` to the floating-point types.